contracts/consensus/CoconsensusI.sol

```diff
@@ -15,26 +15,32 @@ pragma solidity >=0.5.0 <0.6.0;
 // limitations under the License.
 
 /**
- * @title CoconsensusI.
+ * @title Interface of Coconsensus contract.
  *
- * @notice Interface for Coconsensus contract.
+ * @notice Specifies Coconsensus external interfaces to be called from
+ *         other parts.
  */
 interface CoconsensusI {
 
+    /* External Functions */
+
     /**
-     * @notice Gets the anchor address for an metachain id.
+     * @notice Gets the anchor address for a metachain id.
+     *
+     * @param _metachainId A metachain id of an anchor to retrieve.
      *
-     * @return Anchor contract address.
+     * @return anchor_ An anchor contract's address matching to the
+     *                 given metachain id.
      */
-    function getAnchor(bytes32 _metachainId) external returns(address);
+    function getAnchor(bytes32 _metachainId) external returns (address anchor_);
```

Comment:
Maybe this should be `getStateRootProvider`?
Noting this point for discussion later, for now, it fine as we are already using this in code.